api.md

```diff
@@ -287,6 +287,14 @@ in the table.
 to how you handle constraints. A hypertable can contain foreign keys to normal SQL table
 columns, but the reverse is not allowed. UNIQUE and PRIMARY constraints must include the
 partitioning key.
+
+>:WARNING: Running `create_hypertable` with the data migration on a table with foreign key contraints 
```

Comment:
I would change this from a WARNING to a NOTE (or some other keyword like FAQ . any suggestions @solugebefola ).

Reply:
Only special tags are :WARNING: and :TIP: right now.